docs/Solidity.g4

```diff
@@ -358,11 +358,12 @@ assemblyType
 subAssembly
   : 'assembly' identifier assemblyBlock ;
 
+// 'finney' and 'szabo' are no longer supported as denominations by latest Solidity.
```

Comment:
If you keep documenting here perhaps document gwei too :)

Reply:
Documenting ``gwei`` in what way :-)? All the comments I saw in the file are of the form ``<x> is no longer supported by latest Solidity``.. but ``gwei`` *is* still supported.
Happy to add documentation about it, just not sure what exactly you mean :-).